tests/rptest/perf/small_batches_test.py

```diff
@@ -19,31 +19,44 @@ def omb_test(self):
         workload = {
             "name": "SmallBatchesWorkload",
             "topics": 1,
-            "partitions_per_topic": 20000,
+            "partitions_per_topic": 100,
             "subscriptions_per_topic": 1,
-            "consumer_per_subscription": 200,
-            "producers_per_topic": 200,
-            "producer_rate": 150_000,
+            "consumer_per_subscription": 1,
```

Comment:
I guess a lot of consumers aren't really relevant for this test?

Reply:
Yeah, the goal here is to stress the produce path as much as possible. So I set this to reduce to load from the consumers to the lowest possible.